externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/ui/EngineTab.scala

```diff
@@ -110,7 +110,7 @@ case class EngineTab(
     // but if use sbt to build for testing, e.g. docker image, it still uses vanilla jetty classes.
     val shaded = "org.sparkproject.jetty.servlet.ServletContextHandler"
     val vanilla = "org.eclipse.jetty.servlet.ServletContextHandler"
-    if (ClassUtils.classIsLoadable(shaded)) {
+    if (ReflectUtils.isClassLoadable(shaded)) {
```

Comment:
maybe we can inline this method with the following code as replacement
```
    DynClasses.builder()
      .impl("org.sparkproject.jetty.servlet.ServletContextHandler")
      .impl("org.eclipse.jetty.servlet.ServletContextHandler")
      .build()
```

Reply:
It's done in #4879 , by using chaining builders.
https://github.com/apache/kyuubi/pull/4879/files#diff-9a53f1a1af5e3f8d7bebf106722791153acce818fc25e6c61c479f628e9962bcR112

Reply:
it's not enough, let's do it in this one.

- use the chained builder
- inline the code and remove method `loadSparkServletContextHandler`
- rewrite the code behind invoking of `loadSparkServletContextHandler`

Reply:
Addressed.